components/Carousel/Carousel.js

```diff
@@ -1,14 +1,49 @@
 class Carousel {
+    constructor(element) {
+        this.element = element;
+        this.buttonLeft = document.querySelector('.left-button');
+        this.buttonRight = document.querySelector('.right-button');
+        this.buttonLeft.addEventListener('click', () => this.leftShowImage())
+        this.buttonRight.addEventListener('click', () => this.rightShowImage())
+        this.images = Array.from(this.element.querySelectorAll('img'));
+        this.currentIndex = 0;
+        this.rightShowImage();
+    }
 
+    leftShowImage() {
+        this.currentIndex === 0 ? this.currentIndex = 3 : this.currentIndex -= 1;
+        this.images.forEach(img => {
+            img.classList.remove('img-display');
+            img.classList.add('img-none');
+        })
+        this.images[this.currentIndex].classList.remove('img-none');
+        this.images[this.currentIndex].classList.add('img-display');
+        TweenMax.from('.img-none', 0, { opacity: 0 });
+        TweenMax.to('.img-display', 0.2, { opacity: 1 });
+    }
+    rightShowImage() {
+        this.currentIndex === 3 ? this.currentIndex = 0 : this.currentIndex += 1;
+        this.images.forEach(img => {
+            img.classList.remove('img-display');
+            img.classList.add('img-none');
+        })
+        this.images[this.currentIndex].classList.remove('img-none');
+        this.images[this.currentIndex].classList.add('img-display');
+        TweenMax.from('.img-none', 0, { opacity: 0 });
+        TweenMax.to('.img-display', 0.2, { opacity: 1 });
+    }
```

Comment:
Great job in assembling the pieces required of you to bring together a functional carousel.